server/methods.coffee

```diff
@@ -178,9 +178,13 @@ Meteor.methods
     incidents = createIncidentReportsFromEnhancements(enhancements, options)
     incidents = incidents.map (incident) ->
       incident = _.pick(incident, incidentReportSchema.objectKeys())
+    # check for unexpected urls
+    if not options.url.startsWith("promedmail.org/post/")
```

Comment:
Will this fix the security issue?  Could a user still just pass in: `promedmail.org/post/` and nothing else to accomplish the same thing?  Maybe we should check to confirm there is a 5 digit number at the end of the url?